packages/legacy-workbench/src/css/aclmanagement.css

```diff
@@ -225,8 +225,8 @@ input::placeholder {
 }
 
 .acl-tag-deny {
-    background-color: var(--gw-danger-lighter);
-    color: var(--gw-danger-dark);
+    background-color: var(--gw-error-lighter);
+    color: var(--gw-error-dark);
```

Comment:
I think that here is better to be used the tag component properties: --gw-tag-error-background and --gw-tag-error-color

Reply:
Done